classes/item.rb

```diff
@@ -27,13 +27,4 @@ def label=(label)
   def move_to_archive
     @archived = can_be_archived?
   end
-
-  private
-
-  def can_be_archived?
-    current_year = Time.now.year
-    publish_year = Date.parse(@publish_date).year
-    archived = current_year - publish_year
-    archived > 10
-  end
```

Comment:
I think, By mistake, you removed the `Item` class code Please make sure you can't remove the dev branch code for consistency in our code and project.👌👌🤔